File: #Projekt_Daemon/Projekt_Daemon/Models/DestinationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Projekt_Daemon
{
    public class DestinationModel
    {
        public int Id { get; set; }

      
        public int ConfigId { get; set; }

     
        public int SaveType { get; set; }

        public string IP_Address { get; set; }

        public string Username { get; set; }

        public string Password { get; set; }

        public string Path { get; set; }

       
        public string SaveFile { get; set; }
    }
}

File: angularAdminPanel/src/app/models/destination.model.ts
export class Destination {
  id: number;
  configId: number;
  // 0 - local, 1 - NET, 2 - FTP
  saveType: number;
  iP_Address: string | null;
  username: string | null;
  password: string | null;
  path: string;
  saveFile: string;
}
